haskell-src/Concordium/Genesis/Data/P6.hs

```diff
@@ -73,6 +74,9 @@ putVersionedGenesisData gd = do
     putVersion 8
     putGenesisDataV8 gd
 
+parametersToGenesisData :: GenesisParametersV3 'P6 -> GenesisDataP6
+parametersToGenesisData = uncurry GDP6Initial . BaseV1.parametersToState
```

Comment:
In a previous PR this function was removed with the argument that it is not needed. Why is it being reintroduced?

Reply:
This was an oversight as this branched from an old version. `GenesisParametersV3` has now been (re)removed.